lambdas/src/apis/collections/Utils.ts

```diff
@@ -78,7 +78,25 @@ export function createExternalContentUrl(
   return undefined
 }
 
-export function findHashForFile(entity: Entity, fileName: string | undefined) {
+export function createExternalLambdasUrl(urn: string): string {
+  const catalystUrl = baseLambdasServerUrl(process.env.CATALYST_URL) ?? 'http://localhost'
+  return `${catalystUrl}/lambdas/collections/contents/${urn}`
+}
```

Comment:
Is it really necessary to have absolute URLs here? This is usually a bad practice. Servers should not be aware of their domain, doing so prevents flexible configurations using reverse proxies.

Reply:
ERC 721 standard metadata is dynamic and use this external URL to reference the image and thumbnail, I don't see any other option here (before this change we were using the external URL too, but for a content endpoint)

Reply:
could this be used instead?

```ts
  return `https://${req.headers.get('Host')}/lambdas/collections/contents/${urn}`
```

Reply:
yes, excellent 👍